Use tracing instead of log (#307)
twilight-rs/twilight: README.md

}
```

## Note about tracing

> When using tracing you will by default not see logs from libraries that uses the
> `log` crate. You can add that back by using the [`tracing-log`] crate and 
> initalize it like this:

Reviewer: ```suggestion
When using tracing you won't, by default, see logs from any libraries that use the
`log` crate. You can add that back by using the [`tracing-log`] crate and 
initializing it like this:
```
